l10n_be_cooperator/data/mail_template_data.xml

```diff
@@ -13,19 +19,27 @@
         >{{ object.company_id.coop_email_contact or object.company_id.partner_id.email_formatted or user.email_formatted }}</field>
         <field name="partner_to">{{ object.partner_id.id }}</field>
         <field name="subject">Tax Shelter Certificate</field>
+        <field
+            name="description"
+        >Sent to cooperators with their tax shelter certificate in attachment</field>
         <field name="body_html" type="html">
 <div style="margin: 0px; padding: 0px;">
     <p style="margin: 0px; padding: 0px; font-size: 13px;">
         Hello
         <t t-out="object.partner_id.firstname" />,
         <br /><br />
-        You have subscribed to some shares of <t t-out="object.company_id.name" /> on <t t-out="object.declaration_id.fiscal_year" />.
-        You can benefit from the tax shelter, it means a tax reduction of <t t-out="object.declaration_id.tax_shelter_percentage" /> percent on the invested amount.
-        For this you will find in attachment the documents certifying that you've suscribed to <t t-out="object.company_id.name" /> shares.
+        You have subscribed to some shares of <t t-out="object.company_id.name" />
+        during the year <t t-out="object.declaration_id.fiscal_year" />.
+        You can benefit from the tax shelter, it means a tax reduction of
+        <t t-out="object.declaration_id.tax_shelter_percentage" /> percent on
+        the invested amount.
+        For this you will find in attachment the documents certifying that
+        you've suscribed to <t t-out="object.company_id.name" /> shares.
         <br /><br />
-        A dedicated FAQ is coming soon on <t t-out="object.company_id.website" />.
+        A dedicated FAQ is coming soon on <t t-out="object.company_id.website" />
```

Comment:
Is it there yet?

Should the website be wrapped in `<a>` tags, or is this automagically done?